component/core/src/test/java/org/exoplatform/social/core/space/spi/SpaceServiceTest.java

```
@@ -2589,6 +2583,40 @@ private Space createMoreSpace(String spaceName) {
     return space2;
   }
 
+  @SneakyThrows
+  public void testCreateSubspace() {
+
+    SpaceTemplateService spaceTemplateService = getService(SpaceTemplateService.class);
+    SpaceTemplate spaceTemplate = spaceTemplateService.getSpaceTemplates().getFirst();
+    List<String> allowedSubspaceTemplates = new ArrayList<>();
+    allowedSubspaceTemplates.add("1:2");
+    spaceTemplate.setAllowedSubspaceTemplates(allowedSubspaceTemplates);
+    spaceTemplate.setSubspacesMaxLimit(2);
+    spaceTemplateService.updateSpaceTemplate(spaceTemplate);
+
+    Space space = new Space();
+    space.setDisplayName("parentSpace");
+    space.setPrettyName("parentSpace");
+    space.setRegistration(Space.OPEN);
+    space.setDescription("description of space parentSpace");
+    space.setVisibility(Space.PRIVATE);
+    space = spaceService.createSpace(space, ROOT_NAME);
```

This should throw an exception when the `templateId` isn't specified, isn't it ?